BlazarUI/app/stylus/vendor-overrides/bootstrap-overrides.styl

```diff
@@ -15,6 +15,60 @@ $popover-arrow-outer-color ?= $color-calypso-light
 
 $text-muted ?= $color-eerie
 
+$link-color ?= $color-calypso
+$link-hover-color ?= #007A8C
+$text-color ?= $color-heffalump
+
+// // Override bootstrap colors w/ HS styling
+// $gray-base ?= $color-battleship
+// $gray-darker ?= $color-flint
+// $gray-dark ?= $color-eerie
+// $gray ?= $color-koala
+// $gray-light ?= $color-cirrus
+// $gray-lighter ?= $color-gypsum
+//
+// $brand-primary ?= $color-lorax-dark
+// $brand-success ?= $color-oz
+// $brand-info ?= $color-calypso
+// $brand-warning ?= $color-marigold
+// $brand-danger ?= $color-candy-apple
+
+
+// === Table overrides ===
+
+$table-cell-padding ?= 20px
+$table-condensed-cell-padding ?= 10px
+$table-bg ?= $color-olaf
+// No striped tables
```

Comment:
I don't think you should be doing an override to make no stripped tables? Just don't use the table-stripped class